docs/observation/payload.md

```diff
@@ -0,0 +1,43 @@
+# Format of observation payload
+
+Notifications received by OneDrive have a format described on Microsoft's [docs](https://docs.microsoft.com/en-us/graph/api/driveitem-delta?view=graph-rest-1.0&tabs=http) site. When those notifications are received in the `onedrive-change-listener`, they get converted to a portable format before sent to the next stage, e.g.:
+
+```
+{
+  "name": "me--myrepo--master",
+  "params": {
+    "type": "onedrive",
+    "driveId": "...",
+    "owner": "me",
+    "repo": "myrepo",
+    "ref": "master",
+    "mountpoint": {
+      "path": "/ms/",
+      "root": "/myshare"
+    },
+    "changes": [
+      ...
+    ],
+  }
+}
+```
+The `mountpoint` contains the information how paths in the changes array should be translated by replacing a prefix matching `root` with `path`, e.g. for the following change:
+```
+{
+  "id": "...",
+  "relPath": "/myshare/doc.md",
+  "time": "2020-03-03T20:05:23Z",
+  "type": "modified"
+},
+```
+the translated path is `/ms/doc.md`.
+
+When an item in OneDrive is deleted, the associated OneDrive notification does not contain a path. In order to still identify that item, `onedrive-change-listener` computes a hash built from the drive and item id, and includes it in the payloud, e.g.:
+```
+{
+  "id": "...",
+  "sourceHash": "9DGZiYAPqGP5yxlT",
```

Comment:
how is the `sourceHash` calculated?

Reply:
As mentioned in https://github.com/adobe-private/helix-onedrive-listener/pull/52 , isn't needed if `drive-id` and `item-id` are part of the payload.